pandas/tests/base/test_value_counts.py

```diff
@@ -339,3 +339,59 @@ def test_value_counts_object_inference_deprecated():
     exp = dti.value_counts()
     exp.index = exp.index.astype(object)
     tm.assert_series_equal(res, exp)
+
+
+@pytest.mark.parametrize(
+    ("index", "expected_index"),
+    [
+        pytest.param(
+            pd.date_range("2016-01-01", periods=5, freq="D"),
+            pd.date_range("2016-01-01", periods=5, freq="D"),
+        ),
```

Comment:
```suggestion
        [
            pd.date_range("2016-01-01", periods=5, freq="D"),
            pd.date_range("2016-01-01", periods=5, freq="D"),
        ],
```
And can you do the similar pattern below?